easybuild/easyconfigs/b/BUSCO/BUSCO-1.22-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'BUSCO'
+version = '1.22'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://busco.ezlab.org/'
+description = "BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs"
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+
+source_urls = ['http://busco.ezlab.org/v1/files/']
+sources = ['%(name)s_v%(version)s.tar.gz']
+
+checksums = ['86088bbd2128ea04ad9e1b2ebd18201f4c79a48a161ba2593feb12abb8a2d0e2']
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('BLAST+', '2.6.0', versionsuffix),
+    ('HMMER', '3.1b2'),
+    ('AUGUSTUS', '3.2.3', versionsuffix),
+    ('EMBOSS', '6.6.0', '-X11-20170314'),
+]
+
+postinstallcmds = ['chmod +x %(installdir)s/*.py']
+
+sanity_check_paths = {
+    'files': ['BUSCO_v1.22.py'],
```

Comment:
@hajgato please use `%(version)s` rather than hardcoding the version?

Reply:
@boegel done